axelor-stock/src/main/resources/views/StockMoveLine.xml

```diff
@@ -714,34 +714,28 @@
   <action-attrs name="action-stock-move-line-attrs-tracking-number-domain">
     <attribute name="domain" for="trackingNumber"
       expr="eval: &quot;
-              self.product = :product AND (
-               EXISTS (
-                  SELECT sll.id FROM StockLocationLine sll
-                    WHERE sll.trackingNumber.id = self.id
-                    AND sll.detailsStockLocation.id = ${__parent__.fromStockLocation?.id}
-               )
-               OR NOT EXISTS (
-                 SELECT stockLocationLine.id FROM StockLocationLine stockLocationLine
-                   WHERE stockLocationLine.trackingNumber.id = self.id
-               )
-             )
-          &quot;"
+              self.product = :product AND
+	                (self in
+	                    (select stockLocationLine.trackingNumber from StockLocationLine stockLocationLine
+	                    join StockLocation sl on sl.id = stockLocationLine.detailsStockLocation
+	                    WHERE sl.id = ${__parent__.fromStockLocation?.id}) OR
+                     self not in
+                        (select stockLocationLine.trackingNumber from StockLocationLine stockLocationLine
+                        join StockLocation sl on sl.id = stockLocationLine.detailsStockLocation))
+	                &quot;"
```

Comment:
why are we modifying this domain ?

Reply:
The partner that worked on it changed it, but it's similar to the old one.